### PR TITLE
Doc 583 ffc pay report generator update azurite

### PR DESCRIPTION
**Pull request by HarshEviden**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "@azure/data-tables": "13.2.1",
     "@azure/identity": "^4.4.1",
```
Comment (GodsonLeigh) [inline package.json:24]: remove the  ^ we don't want variable version numbers and should be using fixed

**Review by GodsonLeigh — CHANGES_REQUESTED** [review]
small fix required

File: package.json (hunk 1 above)
Comment (HarshEviden) [reply package.json]: Fixed

**Review by GodsonLeigh — APPROVED** [review]
allow

**Pull request merged**